pkg/datapath/linux/route/route_linux.go

```diff
@@ -288,3 +302,41 @@ func Delete(route Route) error {
 
 	return nil
 }
+
+func lookupRule(fwmark int, table int) (bool, error) {
+	rules, err := netlink.RuleList(0)
```

Comment:
Nit: s/0/unix.AF_INET/

Reply:
hmm its actually AF_UNSPEC == 0 here.

Reply:
I'm not sure should we try to avoid importing entire pkgs like unix here just to get a single enum? Maybe I'm thinking to hard about it. Here '0' is fairly clear IMO.

Reply:
Agreed, just maybe add a comment next to it saying `0 =  AF_...` - would help for a reader.